lib/log_decorator/logging.rb

```diff
@@ -21,9 +21,7 @@ def _log_prefix=(prefix)
     end
 
     def self.included(host_class)
-      host_class.extend(ClassMethods) unless host_class.respond_to?(:_log)
-      host_class.instance_logger
-      host_class._log
+      host_class.extend(ClassMethods)
```

Comment:
@hsong-rh This shouldn't be done here. This should be done in the client's initialization code, before `logging` is included in other classes.

Reply:
Ok, moved changes to client side.